MTGToolbox.Repository/MTGToolboxContext.cs

```diff
@@ -1,13 +1,18 @@
-﻿using MTGToolbox.Core;
-using Microsoft.EntityFrameworkCore;
+﻿using Microsoft.EntityFrameworkCore;
+using MTGToolbox.Core;
+using System.Collections.Generic;
 
 namespace MTGToolbox.Repository
 {
     public class MTGToolboxContext : DbContext
     {
         public MTGToolboxContext(DbContextOptions<MTGToolboxContext> options) : base(options)
-        {
+        { }
 
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Deck>()
+                .HasMany(d => (IEnumerable<Card>)d.Cards);
```

Comment:
This is how you tell which implementation to use for a navigation property with an interface. Cards is still an `IEnumerable<ICards>` but with this cast, it'll know to instantiate `Card` when populating this property. I haven't tested since I didn't have a real db to run against but it seemed to have compiled and resolved the mapping fine.